.github/workflows/master.yml

```diff
@@ -51,7 +51,7 @@ jobs:
         run:  mvn compile -Pwin64
 
       - name: Compile Mac
-        run:  mvn compile -Pdarwin64
+        run:  mvn compile -e -Pdarwin64
```

Comment:
Why the -e? is it needed specifically for darwin.
from mvn --help:
 -e,--errors                            Produce execution error messages

Reply:
I added -e because the darwin compile failed and I wanted to see why. Then git did the checks again and everything worked. I like having the extra information in case of a failure.